cluster/addons/fluentd-elasticsearch/fluentd-es-image/td-agent.conf

```diff
@@ -94,20 +94,13 @@
   tag docker
 </source>
 
-<match kubernetes.**>
```

Comment:
Because it was left in by a bad merge. The real match rule is at the bottom of the file.